### ISSUE TITLE
Suggestion: add location or speaking region

### ISSUE THREAD

**Issue by Juice10** [iheanyi/speakers-who-want-a-platform#23]
The world is a big place and not all event organizers have budget to fly all of their speakers from New Zeeland to Amsterdam and back. And not all speakers enjoy sitting in airplanes for a large chunk of their life. Would be nice to list at very least continent or if we would prefer going super detailed; country or city.

We might not want it too granule since it might be one of the reasons why a speaker is not landing a lot of speaking gigs (small villages generally have less of a conference scene than huge sprawling cities). I'd be happy to see what other people think about this.

**Comment by thelouiselee** [comment]
I agree. Just knowing the country would be useful, please.

**Comment by joneshf** [comment]
How should this change be coordinated?

**Comment by Addvilz** [comment]
+20 on this. Smaller, localized, conferences just can't afford sponsoring speaker travel across the planet for everyone.

**Comment by iheanyi** [comment]
Yeah, agreed. I'm wondering how we facilitate this for people who have already submitted their information. Hm.

**Comment by biglovisa** [comment]
We could create a PR for updating the locations and ping the developers that are already in the list to comment with their preferred location, or, have them push a commit to that same branch. 

Just a thought; it would be cool to have this information in a CSV or something instead so that it could be rendered/sorted programmatically instead... 💭

**Comment by Juice10** [comment]
I like @applegrain's suggestion especially if we make it JSON or even better YAML. That would give us some flexibility.
As for those that haven't added this information yet, we can always find some way to facilitate at a later date. Could allow us to experiment a bit before we ask everyone to resubmit their data 3 times.

**Comment by abelards** [comment]
JSON looks nice, some people can point directly to a static content page if they want to make a map, app, or something (RailsGirls and BBL did that).

**Comment by abelards** [comment]
Also would it be possible to add languages? I'm pretty sure many people speak more than one.
Not sure if that should be a separate issue, but problem and solutions look really similar.
Cheers,
